nbr.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "nbr",
+  "release": "11.2-RELEASE",
+  "artifact": "https://github.com/NAKIVO/iocage-plugin-nbr.git",
+  "properties": {
+    "allow_socket_af": "1",
+    "allow_set_hostname": "1",
+    "allow_raw_sockets": "1",
+    "allow_mount_devfs": "1",
```

Comment:
Why are we setting this when we are mounting it below?

Reply:
The Web interface Requiered Tomcat in order to make Tomcat proper to work we need these things up.

Reply:
Those options allow userspace to mount/umount those two filesystems. Since you're already mounting them, why do you care to allow tomcat to unmount them?

Reply:
Same concern with current diffs.  if allow_raw_sockets is for tomcat it shouldn’t be necessary. We have other tomcat plugins that do not require this lowering of security.